test/tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["e2e/**/*", "regressions/**/*", "utils/**/*"],
   "compilerOptions": {
     "allowJs": true,
-    "noEmit": true,
     "noUnusedLocals": true,
     "skipLibCheck": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "moduleResolution": "node16",
```

Comment:
This lets TS respect the `exports` field in other packages (making adding the new package to root tsconfig's paths unnecessary).